docs/ionq/circuits.md

```diff
@@ -96,7 +96,7 @@ these gates are one or two qubit gates which support the `unitary` protocol
 for the gate) there is support for compiling these into API supported gates.
 This conversion may not be optimal, but it does produce a valid API circuit.
 
-This support is given by the `cirq.ionq.IonQAPIDevice` and its
+This support is given by the `cirq_ionq.IonQAPIDevice` and its
 `decompose_operation` method.  On way to use this is to pass the device
```

Comment:
We also need to update this documentation to use the new transformers for compilation (see https://github.com/quantumlib/Cirq/pull/5479). The decompose_operation method as on the device has been deprecated and removed. 

Can you please file a new issue and assign to yourself / someone from IonQ ?

Reply:
Started - I filed an issue at https://github.com/quantumlib/Cirq/issues/5497, can't assign it yet but there's a comment on the ticket requesting it to be assigned to @Cynocracy .